### DISCUSSION TITLE
crud plugins

### DISCUSSION THREAD

**Discussion by hyusetiawan** [graphql-nexus/nexus-prisma#202] [General]
in the "old" nexus we can enable `experimentalCRUDFeature` which will generate a lot of pre-made filtering and more, I wonder if we can do that here in nexus-prisma as well? I see it kind of mentioned here:
https://github.com/prisma/nexus-prisma/issues/15

I searched the docs but could not find anything, is this not possible?

**Comment by ShooTeX** [comment]
looking at the [roadmap](https://nexus.prisma.io/roadmap) it's planned under "longterm"
